Platform/Qemu/SbsaQemu/SbsaQemu.dsc

```diff
@@ -395,10 +399,25 @@ DEFINE NETWORK_HTTP_BOOT_ENABLE       = FALSE
   gArmTokenSpaceGuid.PcdVFPEnabled|1
 
   # System Memory Base -- fixed
-  gArmTokenSpaceGuid.PcdSystemMemoryBase|0x10000000000
+  !if $(ENABLE_RME)
+    #
+    # When RME is enabled the RMM is installed at 0x10000000000
+    # and the system base memory bumped by 1072MB.
+    #
+    gArmTokenSpaceGuid.PcdSystemMemoryBase|0x10043000000
+  !else
+    gArmTokenSpaceGuid.PcdSystemMemoryBase|0x10000000000
+  !endif
 
   # Space for 32 stacks
-  gArmPlatformTokenSpaceGuid.PcdCPUCoresStackBase|0x1000007c000

```

Comment:
I wonder can we have something like this:

`gArmPlatformTokenSpaceGuid.PcdCPUCoresStackBase|gArmTokenSpaceGuid.PcdSystemMemoryBase+0x7c000`

This would remove a need for extra "if".

Reply:
Checked. Such definition works so code can be a bit simpler.

Reply:
@hrw https://github.com/tianocore/edk2/blob/master/ArmPlatformPkg/Sec/AArch64/ModuleEntryPoint.S#L33
It's a weird pattern, but it adds up.

Ultimately, I guess we should only reserve the one stack these days? @ardbiesheuvel?